Userland/Libraries/LibTLS/TLSv12.h

```diff
@@ -161,11 +163,11 @@ struct Options {
         { HashAlgorithm::SHA512, SignatureAlgorithm::RSA },
         { HashAlgorithm::SHA384, SignatureAlgorithm::RSA },
         { HashAlgorithm::SHA256, SignatureAlgorithm::RSA },
-        { HashAlgorithm::SHA1, SignatureAlgorithm::RSA });
-    OPTION_WITH_DEFAULTS(Vector<SupportedGroup>, elliptic_curves,
-        SupportedGroup::X25519,
-        SupportedGroup::SECP256R1,
-        SupportedGroup::X448)
```

Comment:
These should not be removed, as they are used for key exchange, not certificate signing